static/app/utils/replays/replayReader.tsx

```diff
@@ -685,29 +685,25 @@ export default class ReplayReader {
   });
 
   getWebVitalFrames = memoize(() => {
-    if (this._featureFlags?.includes('session-replay-web-vitals')) {
```

Comment:
typeerror w/ `_featureFlags` -- how was it being populated?

Reply:
cleaned it up too